test/common/index.js

```diff
@@ -568,9 +568,24 @@ exports.canCreateSymLink = function() {
   return true;
 };
 
+function getCallSite(level = 2) {
+  const originalStackFormatter = Error.prepareStackTrace;
+  Error.prepareStackTrace = (err, stack) =>
+    `${stack[level].getFileName()}:${stack[level].getLineNumber()}`;
+  const err = new Error();
+  Error.captureStackTrace(err, level);
+  // the way V8 Error API works, the stack is not
+  // formatted until it is accessed
+  err.stack;
+  Error.prepareStackTrace = originalStackFormatter;
+  err.stack;
+}
```

Comment:
I don't understand how this works. There is no return.

Reply:
Stupid mistake. Fixed and test added.